multinet/db.py

```diff
@@ -153,6 +153,7 @@ def rename_workspace(old_name: str, new_name: str) -> None:
     workspace_mapping.cache_clear()
 
 
+# TODO: Add internal function for deleting workspace
```

Comment:
Why?

Reply:
Should've made this more clear, what I meant was that there should be a function that deletes a workspace given it's `internal` name. It seems like a generally useful function, only really to be used in edge cases or worst case scenarios. I thought of it when looking at the [`test_workspace_rename`](https://github.com/multinet-app/multinet-server/blob/acc8864e8f7f114b15b5f380ce29540955957088/test/test_workspace.py#L84) test. If we had this internal function, we could use this in a way similar to the `handled_workspace` fixture.

Reply:
Gotcha. Can you file an issue to do this and remove the comment?

Reply:
435b35c